tools/Localizations/Sources/Localizations/ExportTask.swift

```diff
@@ -67,6 +61,30 @@ struct ExportTask {
         try! task.run()
         task.waitUntilExit()
     }
+    
+    // Get Locales that are in the Xcode project
+    private func getProjectLocales() -> (Array<String>) {
+    
+        var localesList:[String] = []
+
+        let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: xcodeProjPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path)
+        let filePaths = blockzillaFolder?.allObjects as! [String]
+        let textFilePaths = filePaths.filter{$0.contains(".lproj")}
+        for txt in textFilePaths {
```

Comment:
Nothing wrong with splitting it up. You can make it more compact with

```
filePaths.filter { $0.contains(".lproj") }.forEach { txt in
   ...
}
```

We don't really have guidelines.